src/GridView/HierarchyStepper.js

```diff
@@ -69,18 +63,27 @@ class HierarchyStepper extends Component {
 			<Stepper
 				activeStep={this.state.activeStep}
 				className={css(this.props.styles, AppStyles.elementMarginTop)}>
-				{this.props.hierarchy.map((hierarchyLayer, index) => {
+
+				{this.state.hierarchy.map((hierarchyLayer, index) => {
 					return (
 						<Step key={index}>
-							{hierarchyLayer.map((entityType, index) => {
-								return this.displayStepLabel(entityType, index);
-							})}
+							{hierarchyLayer.map((entityType, index) => {return(
+								this.displayStepLabel(entityType, index)
+							);})}
 						</Step>
 					);
 				})}
 			</Stepper>
-		);
+		)
 	}
 }
 
-export default HierarchyStepper;
\ No newline at end of file
+
+export default connect.defaults({fetch: ConnectionComponent.switchFetch})(props => ({
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "props". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://bpt-lab.org/sonarqube/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)